library/src/test/java/com/datatorrent/lib/io/jms/JMSTransactionableStoreTestBase.java

```diff
@@ -157,6 +187,35 @@ public void storeRetreiveTransactionTest()
     deleteOperator();
   }
 
+  /**
+   * Creates two operators with different operatorId and same appId to test correct functionality of storing and
+   * retrieving windowId with message selector
+   */
+  //@Ignore
+  @Test
+  public void twoOperatorsStoreRetrieveWithMessageSelectorTransactionTest()
+  {
+    createOperator();
+
+    outputOperator.beginWindow(0L);
+    outputOperator.endWindow();
+
+    //Create fresh operator context
+
+    JMSStringSinglePortOutputOperator outputOperator2 = createOperator2();
+    outputOperator2.beginWindow(1L);
+    outputOperator2.endWindow();
+
+    long windowIdOp = outputOperator.getStore().getCommittedWindowId(APP_ID, OPERATOR_ID);
+    Assert.assertEquals(0L, windowIdOp);
+
+    long windowIdOp2 = outputOperator2.getStore().getCommittedWindowId(APP_ID, OPERATOR_2_ID);
+    Assert.assertEquals(1L, windowIdOp2);
+
+    deleteOperator();
```

Comment:
refactor deleteOperator to take operator as an argument and use it for outputOperator and outputOperstor2